src/test/java/com/zendesk/maxwell/row/RowMapTest.java

```diff
@@ -51,7 +52,7 @@ public void testGetRowIdentity() {
 
 		RowMap rowMap = new RowMap("insert", "MyDatabase", "MyTable", TIMESTAMP_MILLISECONDS, pKeys, POSITION);
 
-		rowMap.putData("id", "9001");
+		rowMap.putData("id", 9001);
```

Comment:
why should these tests change?  doesn't seem right to me...

Reply:
The test was changed to show how the "primary_key" is outputted when dealing with non string values. ie. "primary_key": [9001, "foo"] vs. "primary_key": ["9001", "foo"].